src/Microsoft.DotNet.Interactive.SqlServer/MsSqlServiceClient.cs

```diff
@@ -158,10 +156,10 @@ public async Task<QueryExecuteSubsetResult> ExecuteQueryExecuteSubsetAsync(Query
 
         public async Task SendTextChangeNotificationAsync(Uri ownerUri, string newText, string oldText)
         {
-            var oldTextLines = oldText.Split('\n');
+            var oldTextLines = oldText.Split(Environment.NewLine);
```

Comment:
It's a common scenario that the compute and the editor are on different machines with different newlines, or that a notebook file was written with `\r\n` newlines but is being evaluated with `\n` newlines. So the split should handle both.

Reply:
in ntoebook parsing we use this logic 
`s.Split('\n').Select(l => l.EndsWith('\r') ? l[0..^1] : l);`

Reply:
Fixed with Diego's suggestion.